integration/Makefile

```diff
@@ -8,13 +8,10 @@ test:
 
 # Run the integration test suite
 .PHONY: integration-test
-integration-test: access-tests common-tests execution-tests verification-tests collection-tests epochs-tests network-tests consensus-tests 
+integration-test: access-tests common-tests execution-tests verification-tests collection-tests epochs-tests network-tests consensus-tests
 
-# NOTE: Currently skipping collection and consensus tests due to them not passing properly on CI
-# NOTE: Currently skipping all integration tests while replacing modules for consensus V2
 .PHONY: ci-integration-test
-
-ci-integration-test: access-tests common-tests execution-tests epochs-tests verification-tests network-tests # collection-tests # consensus-tests
+ci-integration-test: access-tests common-tests execution-tests epochs-tests verification-tests network-tests # collection-tests consensus-tests
```

Comment:
Keep the collection-tests and consensus tests disabled. 

They are passing on local but failing on CI. Will address the flakiness in separate PR